### PR TITLE
boot-menu: Slightly reformat & add pang12

### LINKED ISSUE #1116: BIOS key for pang12 needs to be added to Access BIOS article
https://support.system76.com/articles/boot-menu

This one might be a little tricky since it's `Esc` for BIOS, and the boot menu is also in BIOS, meaning it's almost the same as open firmware. We might have to call it out specifically by model, along with any future models in this line that share the same BIOS.

### PR DESCRIPTION
**Pull request by jacobgkau**
Closes #1116.

image

### CODE REVIEW

File: content/boot-menu.md [hunk 1]
```diff
@@ -16,15 +16,15 @@ section: software
...
-| Laptop - Open Firmware | ESC      | ESC(select one time boot option) |
-| Laptop - Proprietary   | F2       | F7                               |
-| Older Laptops          | Depends on the system | F1                  |
-| Thelio                 | Del      |  F8 or F12                       |
-| Meerkat                | F2       | F10                              |
+The BIOS or boot menu can be accessed by holding specific keys down during boot. Refer to the table below for the keys on each model:
+
+| Model                                                       | BIOS key | Boot Menu key                     |
+|:-----------------------------------------------------------:|:--------:|:---------------------------------:|
+| Laptops:<br/>All Open Firmware models<br/>Pangolin (pang12) | ESC      | ESC (select one-time boot option) |
```
Comment (leviport) [inline content/boot-menu.md:23]: I wonder if the `(select one-time boot option)` note should be tweaked to account for pang12's boot menu:

IMG_20230320_155402_417

I'm thinking maybe something like, "(select one-time boot option in open firmware, or use the boot override menu in "save & exit" on pang12)," but it's awfully wordy.

**Comment by jacobgkau** [comment]
Is this better?

image

File: content/boot-menu.md (hunk 1 above)
Comment (jacobgkau) [reply content/boot-menu.md]: Is this better?

image

Comment (jacobgkau) [reply content/boot-menu.md]: I guess we might want the page that it's on for proprietary.

**Comment by leviport** [comment]
Yeah that looks good! I think the case should be consistent though. Either make (Open Firmware) lowercase or capitalize (proprietary firmware).

File: content/boot-menu.md (hunk 1 above)
Comment (jacobgkau) [reply content/boot-menu.md]: Here's another option:

image

Is that clear enough?

**Comment by jacobgkau** [comment]
> Yeah that looks good! I think the case should be consistent though. Either make (Open Firmware) lowercase or capitalize (proprietary firmware).

Open Firmware is a product name, proprietary firmware is a description.

**Comment by leviport** [comment]
Ah, fair, I hadn't thought of it as a product name.

**Review by leviport — APPROVED** [review]

**Comment by jacobgkau** [comment]
I will need to either allow the `<br>` tag in the Markdown linter or find another way to break the line within the table cells.

**Comment by jacobgkau** [comment]
I allowed this in the linter because Markdown doesn't seem to have a way to include line breaks within table cells by default (some specific flavors of Markdown have their own ways of doing this, but I didn't quickly find one that works with the Nuxt parser that we're using.)

**Comment by jacobgkau** [comment]
This is what it looks like after the last commit:

image

My first instinct was that the code tags made it look to busy, but it looks okay to me after considering it further. I can revert the last commit if other people don't like it.

**Review by ahoneybun — APPROVED** [review]
LGTM

**Pull request merged**